arangod/Agency/Node.h

```diff
@@ -336,6 +336,20 @@ class Node {
   /// @brief Get double value (throws if type NODE or if conversion fails)
   double getDouble() const;
 
+  template<typename T>
+  auto getNumberUnlessExpiredWithDefault() -> T {
+    if (ADB_UNLIKELY(!lifetimeExpired())) {
```

Comment:
```suggestion
    if (ADB_LIKELY(!lifetimeExpired())) {
```

Reply:
https://en.cppreference.com/w/cpp/language/attributes/likely

Reply:
Unfortunately that is part of the standard from C++20 on only.

Reply:
Oh I forgot that it is not available here:)